packages/design-system/src/components/search/search.js

```diff
@@ -178,9 +180,9 @@ export const Search = ({
 
   const handleClearInput = useCallback(() => {
     inputState.set(undefined);
-    onMenuItemClick?.(null, { label: '', value: '' });
+    onClear();
```

Comment:
If removing this happens to break anything else consuming this, I'm fine leaving it in with the `onClear` event ie.
```
onMenuItemClick?.(null, { label: '', value: '' });
onClear();
```

Reply:
I checked all the search inputs I could find. Seemed to not break anything. And I didn't get any failing tests, so 🤷

Reply:
if memory serves this was added because of edge cases in Storybook, so just make sure you also update the Search storybook components so that doesn't break